Comments (hash-sign) in flake8.ignore parameters lead to incorrect behaviour

Hi, here is my `setup.cfg` content:

```ini
[flake8]
enable-extensions = G
exclude = .git, .venv
ignore =
    A003 # 'id' is a python builtin, consider renaming the class attribute
    W503 # line break before binary operator
    D100 # Missing docstring in public module
    D102 # D102 Missing docstring in public method
    D103 # Missing docstring in public function
    D104 # Missing docstring in public package
    D107 # Missing docstring in __init__
    S101 # assert detected
    S105 # Possible hardcoded password
    S106 # Possible hardcoded password
    S107 # Possible hardcoded password
    D106
    C408 # Unnecessary dict call
    D401
    X100
```

This configuration is valid and flake8 works as expected, but nether `flakehell baseline` nor `flakehell lint` do not produce any valuable output. The reason can be found in the debug logs:

```text
flake8.options.aggregator MainProcess    665 DEBUG    Overriding default value of (['E121', 'E123', 'W503', 'E704', 'W504', 'E24', 'E126', 'E226']) for "ignore" with (['A003', '#', "'id'", 'is', 'a', 'python', 'builtin', 'consider', 'renaming', 'the', 'class', 'attribute', 'W503', '#', 'line', 'break', 'before', 'binary', 'operator', 'D100', '#', 'Missing', 'docstring', 'in', 'public', 'module', 'D102', '#', 'D102', 'Missing', 'docstring', 'in', 'public', 'method', 'D103', '#', 'Missing', 'docstring', 'in', 'public', 'function', 'D104', '#', 'Missing', 'docstring', 'in', 'public', 'package', 'D107', '#', 'Missing', 'docstring', 'in', '__init__', 'S101', '#', 'assert', 'detected', 'S105', '#', 'Possible', 'hardcoded', 'password', 'S106', '#', 'Possible', 'hardcoded', 'password', 'S107', '#', 'Possible', 'hardcoded', 'password', 'D106', 'C408', '#', 'Unnecessary', 'dict', 'call', 'D401', 'X100'])
```

Seems like all errors are ignored. Any comments? Thank you.

@orsinium I'm still trying to adopt the library and force it to do what flake8 already did but to no avail.

Here are my steps:

```bash
# ... virtual env creation with 3.7.2
# (installed 0.3.0)
$ pip install flakehell 

$ cat > script.py
def hello(name):
    print('World!')

text = "Very long long long string"

$ cat > pyproject.toml
[tool.flakehell]
max_line_length = 20

[tool.flakehell.plugins]
pycodestyle = ["+*"]
pylint = ["+*"]

$ flake8 --max-line-length=20
./script.py:1:11: W0613 Unused argument 'name'
./script.py:4:1: E305 expected 2 blank lines after class or function definition, found 1
./script.py:4:21: E501 line too long (35 > 20 characters)

$ flakehell lint script.py
script.py:1:11: W0613 Unused argument 'name' [pylint]

$ flakehell plugins
NAME        | VERION | CODES  | RULES
mccabe      | 0.6.1 | C90    |
pycodestyle | 2.5.0 | E, W   | +*
pyflakes    | 2.1.1 | F      |
pylint      | 2.4.3 | C, E, F, I, R, W | +*

$ flakehell missed
<nothing>
```

What have I missed? How can I get the same results as produced by flake?

--- a/flakehell/_patched/_checkers.py
+++ b/flakehell/_patched/_checkers.py
@@ -87,7 +87,16 @@ class FlakeHellCheckersManager(Manager):
         """
         # self.run_serial()
         results_reported = results_found = 0
+        showed = set()
         for checker in self.checkers:
+            if not checker.results:
+                continue
+
+            # IDK why we have duplicates but let's fight it
+            if checker.display_name in showed:
+                continue
+            showed.add(checker.display_name)
+
             results = sorted(checker.results, key=lambda tup: (tup[1], tup[2]))
             with self.style_guide.processing_file(checker.filename):
                 results_reported += self._handle_results(
@@ -99,8 +108,6 @@ class FlakeHellCheckersManager(Manager):
         return (results_found, results_reported)
 
     def _handle_results(self, filename: str, results: list, check: dict) -> int:
-        if not results:
-            return 0
         plugin_name = get_plugin_name(check)
         rules = get_plugin_rules(
             plugin_name=plugin_name,

--- a/flakehell/_patched/_checkers.py
+++ b/flakehell/_patched/_checkers.py
@@ -145,8 +145,8 @@ class FlakeHellFileChecker(FileChecker):
         super().__init__(filename=filename, checks=checks, options=options)
 
         # display_name used in run_parallel for grouping results.
-        # Flake8 groups by filename, we need to group also by plugin name
-        self.display_name = (get_plugin_name(check), filename)
+        # Flake8 groups by filename, we need to group also by the check
+        self.display_name = (get_plugin_name(check), check['name'], filename)
 
     def __repr__(self):
         return '{name}({plugin}, {filename})'.format(

